back/app/controllers/web_api/v1/admin_publications_controller.rb

```diff
@@ -13,6 +13,16 @@ def index
       .order(:ordering)
     @publications = paginate @publications
 
+    if params[:search].present?
+      LogActivityJob.perform_later(
```

Comment:
Is this really a worthy action to log? :sweat_smile: We have most of the info in the request logs anyway (except current user)

Reply:
@jinjagit?

Reply:
If we have the info elsewhere, then sure, remove it.

Sorry, Lucas, my mistake for advising to add this.

Reply:
No problem at all. I don't actually know much about how we log requests so it's good to learn the various options regardless

Reply:
> We have most of the info in the request logs anyway

Aren't they deleted after deployment? I don't think we have central storage yet, do we? If not, I would maybe preserve this logging. But definitely not that crucial.

Reply:
@nTraum Are the request logs persisted?

Reply:
> Aren't they deleted after deployment?

Yes, that's why I would add the logging. It gets persisted and can, therefore, be used for later study of how the feature was used.

Reply:
@lilkraftwerk I wouldn't let this block you. We can always add it back in if needed, if you decide to remove it.

Reply:
Yeah, not a blocker. Let's leave it in if you think it makes sense. I just cannot imagine an issue where this will be used, but it doesn't hurt.

Reply:
Advertising our success? 😄 "X% of users on Y% of platforms have used project search since it was introduced"